src/queues/EventRedactionQueue.ts

```diff
@@ -64,6 +64,7 @@ export class RedactUserInRoom implements QueuedRedaction {
  * This is a queue for events so that other protections can happen first (e.g. applying room bans to every room).
  */
 export class EventRedactionQueue {
+    /** This map is indexed by roomId and its values are a list of redactions waiting to be processed for that room. */
```

Comment:
Nit: newline after `/**` and before `*/`. There's another format for single-line doc comments, but I forgot what it looks like. Maybe `//*`?